src/core/dev_api/validation_util.hpp

```diff
@@ -77,14 +77,13 @@ Tensor make_tensor_of_min_value(const element::Type_t et);
 ///
 /// \return true if auto padding was applied successfully (all needed informations such as
 ///         spatial dims are known), false otherwise.
-OPENVINO_API
-bool try_apply_auto_padding(const PartialShape& image_shape,
-                            const Shape& filter_shape,
-                            const Strides& filter_strides,
-                            const Strides& filter_dilations,
-                            const op::PadType pad_type,
-                            CoordinateDiff& padding_above,
-                            CoordinateDiff& padding_below);
+OPENVINO_API bool try_apply_auto_padding(const PartialShape& image_shape,
```

Comment:
Mark this function as deprecated. It should not be used the as `shape_infer` for operator should do it. The plugin maybe there is single operator should be updated and use `shape_infer`.
Not sure if this can be removed in 2024 but should be removed soon.